README.md

```diff
@@ -8,7 +8,16 @@
 ## Working Changes
 [Read the working changes to the spec](spec.md)
 
-These changes have been accepted by the working group for the next version of the specification, however are still subject to change.
+These changes have been accepted by the working group for the next version of
+the specification, however are still subject to change.
+
+## Alpha Features
+
+We release some features at an alpha level to master before they become an
```

Comment:
Minor thing, but to someone less geeky than us it might be better to call out some of these words (like "master") as having special meaning.  For example:

Some features are released as "alpha level" features into the `master` branch before
they become an official part of the API specification.